src/DIRC_geo.cpp

```diff
@@ -108,46 +108,40 @@ static Ref_t createDetector(Detector& desc, xml_h e, SensitiveDetector sens)
 
   // Lens variables
   xml_comp_t xml_lens    = xml_module.child(_Unicode(lens));
-  double     lens_height = getAttrOrDefault(xml_lens, _Unicode(height), 50 * mm);
   double     lens_shift  = getAttrOrDefault(xml_lens, _Unicode(shift), 0 * mm);
-  // double lens_width = getAttrOrDefault(xml_lens, _Unicode(width), 25 * mm);
+  double lens_width = getAttrOrDefault(xml_lens, _Unicode(width), 35 * mm);
   double lens_thickness = getAttrOrDefault(xml_lens, _Unicode(thickness), 12 * mm);
   double lens_r1        = getAttrOrDefault(xml_lens, _Unicode(r1), 62 * mm);
   double lens_r2        = getAttrOrDefault(xml_lens, _Unicode(r2), 36 * mm);
 
   // Lens construction
   // FIXME avoid negative impact of booleans by putting lens inside box
 
-  // lens_min_thickness is distance from face to r1, and from r1 to r2 at lens top edge
-  double lens_min_thickness = 0.5 * mm;
-  double layer01            = 1.0 * lens_min_thickness;
-  double layer12            = 2.0 * lens_min_thickness;
+  // 3-layer spherical lens ---
 
-  // ztrans1 and ztrans2 are the z shifts of the lens center for r1 and r2
-  double ztrans1 = lens_thickness / 2. + sqrt(lens_r1 * lens_r1 - lens_height / 2. * lens_height / 2.) - layer01;
-  double ztrans2 = lens_thickness / 2. + sqrt(lens_r2 * lens_r2 - lens_height / 2. * lens_height / 2.) - layer12;
+  double cr2 = sqrt(lens_width * lens_width / 4. + bar_heigth * bar_heigth / 4.);
```

Comment:
Helps to spell it correctly...
```suggestion
  double cr2 = sqrt(lens_width * lens_width / 4. + bar_height * bar_height / 4.);
```